modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -400,6 +400,9 @@
 				m.libraryPath,
 				m.extensionsToDeleteOnClean,
 				m.executablePath,
+
+				-- Linux
+				m.linuxMultiProcNumber
```

Comment:
This needs tests.

Reply:
I was a bit in doubt as I've chosen the number 8 for it which is a bit arbitrary, it's better than not testing anything I suppose